WordPress/Resources/en.lproj/Localizable.strings

```diff
@@ -4365,6 +4370,37 @@
 /* Menu item title to hide the card for now and show it later. */
 "jetpack.menuCard.remindLater" = "Remind me later";
 
+/* Jetpack Plugin Modal footnote */
+"jetpack.plugin.modal.footnote" = "By setting up Jetpack you agree to our %@";
+
+/* Jetpack Plugin Modal primary button title */
+"jetpack.plugin.modal.primary.button.title" = "Install the full plugin";
+
+/* Jetpack Plugin Modal secondary button title */
+"jetpack.plugin.modal.secondary.button.title" = "Contact Support";
+
+/* The 'full Jetpack plugin' string in the subtitle */
+"jetpack.plugin.modal.subtitle.jetpack.plugin" = "full Jetpack plugin";
+
+/* Jetpack Plugin Modal (multiple plugins) subtitle with formatted texts.
+%1$@ is for the site name, and %2$@ for 'full Jetpack plugin' in bold style. */
+"jetpack.plugin.modal.subtitle.plural" = "%1$@ is using individual Jetpack plugins, which don't support all features of the app yet.
+
+Please install the %2$@ to use the app with this site.";
+
+/* Jetpack Plugin Modal (single plugin) subtitle with formatted texts.
+%1$@ is for the site name, %2$@ for the specific plugin name,
+and %3$@ is for 'full Jetpack plugin' in bold style. */
+"jetpack.plugin.modal.subtitle.singular" = "%1$@ is using the %2$@ plugin, which doesn't support all features of the app yet.
+
+Please install the %3$@ to use the app with this site.";
```

Comment:
I was surprised by these empty lines. However, `genstrings` generated this file and the linting worked so I guess we're good.